epitome/dataset.py

```diff
@@ -267,25 +272,56 @@ def get_y_indices_for_target(matrix, targetmap, target):
         return np.copy(matrix[:,targetmap[target]])
 
     @staticmethod
-    def get_data_dir(data_dir=None):
+    def contains_required_files(data_dir):
+        # make sure all required files exist
+        required_paths = [os.path.join(data_dir, x) for x in REQUIRED_FILES]
+        return np.all([os.path.exists(x) for x in required_paths])
+
+    @staticmethod
+    def list_genome_assemblies():
+        return ", ".join(EPITOME_GENOME_ASSEMBLIES)
+
+    @staticmethod
+    def get_epitome_user_path():
+        return os.path.join(os.path.expanduser('~'), '.epitome')
+
+    @staticmethod
+    def get_data_dir(data_dir=None, assembly=None):
         '''
         Loads data processed from data/download_encode.py. This will check that all required files
-        exist.
+        exist. If both data_dir and assembly are set, it will return the data_dir with the specified
+        assembly. If only the assembly is set, it will return the default data_dir with the specified
+        assembly. If only the data_dir is set, it will just return the data_path. If neither data_dir
+        nor assembly are set, it will return the default data_dir with the default assembly.
 
         :param str data_dir: Directory containing data.h5 file saved in data/download_encode.py script.
         :return: directory containing data.h5 file
         :rtype: str
         '''
-
-        if not data_dir:
-            data_dir = GET_DATA_PATH()
-            download_and_unzip(S3_DATA_PATH, os.path.dirname(data_dir))
-
-        # make sure all required files exist
-        required_paths = [os.path.join(data_dir, x) for x in  REQUIRED_FILES]
-        assert(np.all([os.path.exists(x) for x in required_paths]))
-
-        return data_dir
+        default_data_dir = os.path.join(EpitomeDataset.get_epitome_user_path(), 'data')
+
+        if (data_dir is not None) and (assembly is not None):
+            epitome_data_dir = os.path.join(data_dir, assembly)
+        elif (assembly is not None):
+            epitome_data_dir = os.path.join(default_data_dir, assembly)
+        elif (data_dir is not None):
+            epitome_data_dir = data_dir
+        else:
+            default_assembly = 'hg19'
```

Comment:
update this to hg38, make a constant at the beginning of the file so its obvious